### DISCUSSION TITLE
Sticky Table Header

### DISCUSSION THREAD

**Discussion by ldvlgr** [twilio-labs/paste#3149] [General] [answered]
Using a [Table with a Sticky Header](https://paste.twilio.design/components/table#table-with-sticky-header) in a Flex plugin causes it to overlay the expanded Flex Menu as shown here.
<img width="741" alt="AgentAdminTable" src="https://user-images.githubusercontent.com/75390074/231170457-ab2759e9-062d-467e-a679-9bbe0a1610cd.png">
<img width="760" alt="StickyHeader" src="https://user-images.githubusercontent.com/75390074/231170460-730eb71c-b74f-40c1-8179-a4b4aa817d6d.png">

**Comment by TheSisb** [comment, marked as answer]
Hi @ldvlgr,

This is one of those situations where the z-index set on the sticky header is larger than the z-index set on the Flex sidebar.  We use our smallest non-zero [z-index token](https://paste.twilio.design/tokens/list#z-indices), `$z-index-10`, which translates to a value of `10` on the table header. We would recommend applying a token on the Flex sidebar set to z-index `$z-index-20` or larger to fix the issue. Since we do not maintain the Flex product, we cannot do this ourselves. 

Hope this helps!
